readme.md

```diff
@@ -1,4 +1,4 @@
-# eslint-plugin-unicorn [![Build Status](https://github.com/sindresorhus/eslint-plugin-unicorn/workflows/CI/badge.svg)](https://github.com/sindresorhus/eslint-plugin-unicorn/actions) [![Coverage Status](https://coveralls.io/repos/github/sindresorhus/eslint-plugin-unicorn/badge.svg?branch=master)](https://coveralls.io/github/sindresorhus/eslint-plugin-unicorn?branch=master)
+# eslint-plugin-unicorn [![Build Status](https://github.com/sindresorhus/eslint-plugin-unicorn/workflows/CI/badge.svg)](https://github.com/sindresorhus/eslint-plugin-unicorn/actions?query=branch%3Amaster) [![Coverage Status](https://coveralls.io/repos/github/sindresorhus/eslint-plugin-unicorn/badge.svg?branch=master)](https://coveralls.io/github/sindresorhus/eslint-plugin-unicorn?branch=master)
```

Comment:
```suggestion
# eslint-plugin-unicorn [![Build Status](https://github.com/sindresorhus/eslint-plugin-unicorn/workflows/CI/badge.svg?branch=master&event=push)](https://github.com/sindresorhus/eslint-plugin-unicorn/actions?query=branch%3Amaster+workflow%3ACI+event%3Apush) [![Coverage Status](https://coveralls.io/repos/github/sindresorhus/eslint-plugin-unicorn/badge.svg?branch=master)](https://coveralls.io/github/sindresorhus/eslint-plugin-unicorn?branch=master)
```

Actually, it should be this. PRs should not affect it.

Reply:
Too bad it's not possible to link directly to the latest run. You should definitely send a feature request to GitHub about that.

Reply:
Shouldn't master alway push?

Reply:
I thought there might be, but I can't come up with anything specific, so I've dropped that part now.